src/main/java/uk/gov/justice/digital/hmpps/whereabouts/services/court/CourtService.kt

```diff
@@ -78,7 +77,8 @@ class CourtService(
         post = postEvent?.let { toAppointment(postEvent.eventId, HearingType.POST, specification) }
       )
     )
-    trackVideoLinkBookingCreated(persistentBooking, specification, mainEvent.agencyId)
+    videoLinkBookingRepository.flush() // Force ids to be populated
```

Comment:
I am surprised this is needed, given that it is all done in a single transaction